external/README

```diff
@@ -73,3 +73,24 @@ PS3EYEDriver
 PSEye userspace driver based on libusb + the Linux driver
 by Eugene Zatepyakin, modified by Thomas Perl
 from https://github.com/thp/PS3EYEDriver
+
+pthreads-w32
+============
+
+A Windows implementation of posix-threads.
+https://www.sourceware.org/pthreads-win32/
+
+msvc-support
+============
+
+Work-arounds for MSVC's non-compliant C and a few unix-like libraries.
+These were taken from various sources.
+
+unistd.h is from http://stackoverflow.com/questions/341817/is-there-a-replacement-for-unistd-h-for-windows-visual-c
+There is no license.
```

Comment:
If there would be no license, we wouldn't be allowed to include it.

For stackoverflow, generally this applies: `user contributions licensed under cc by-sa 3.0 with attribution required`

Please spend some time figuring out from where the code came from and specify the license that applies or rewrite that part from scratch; we can't accept code without a license.